test/runner/file-match.test.ts

```diff
@@ -0,0 +1,76 @@
+import { globMatchFiles } from "../../src/util";
+
+let wrappedGlobMatchFiles = async (filePatterns: string[]) => {
+    let results = await globMatchFiles(filePatterns);
+    return results.map((result) => result.split(__dirname + "/resources/")[1]);
+};
+
+describe("globMatchFiles", () => {
+    let spy: jest.SpyInstance;
+
+    beforeEach(() => {
+        spy = jest.spyOn(process, "cwd");
+        spy.mockReturnValue(__dirname + "/resources");
+    });
+
+    afterEach(() => {
+        spy.mockRestore();
+    });
+
+    it("Finds all .test.brs files when not given patterns", async () => {
+        let results = await wrappedGlobMatchFiles([]);
+        expect(results).toEqual([
+            "fly-you-fools.test.brs",
+            "bar/bar-suffix.test.brs",
+            "bar/bar.test.brs",
+            "bar/prefix-bar.test.brs",
+            "foo/main.test.brs",
+            "foo2/main.test.brs",
+        ]);
+    });
+
+    it("Partially matches on filename suffix when given a .brs extension", async () => {
+        let results = await wrappedGlobMatchFiles(["bar.test.brs"]);
+        expect(results).toEqual([
+            "bar/bar.test.brs",
+            "bar/prefix-bar.test.brs",
+        ]);
```

Comment:
Including `prefix-bar.test.brs` isn't quite what I expected, but I think I provide the full path to files when I `jest /path/to/some/file.test.js` so perhaps this matches the `jest` behavior? 🤔

Reply:
Yep, this does match `jest` behavior (and yes, I realize we're `roca` not `rest`, but I still like a lot of what `jest` does 😉).

For example, in this repo's tests:
![Screen Shot 2021-04-20 at 11 49 50 AM](https://user-images.githubusercontent.com/11684087/115448436-8e706e80-a1ce-11eb-8223-496b89526a76.png)